misc/pytorch_toolkit/text_recognition/init_venv_.sh

```diff
@@ -12,7 +12,7 @@ if [[ -e venv ]]; then
 fi
```

Comment:
please rename it to 'init_venv.sh' otherwise tests are not run.

Reply:
renamed